docs/release-notes/highlights-7.7.0.asciidoc

```diff
@@ -90,6 +90,17 @@ The distance filter is in addition to the tools for filtering data by custom sha
 [role="screenshot"]
 image:release-notes/images/7-7-maps-distance-filter.png[Distance filters in Elastic Maps]
 
+[float]
+[[file-uploader-highlights]]
+=== {data-viz} now recommends {filebeat} config
+
+experimental[] If you upload a file to the *{data-viz}*, it now has the ability to recommend a {filebeat} 
```

Comment:
Could you tighten up the first sentence a bit? I'm getting stuck on the `it` part. Maybe something like:

When you upload a file to *{data-viz}*, a {filebeat} configuration is recommended for you to use.